prov/usnic/src/usdf.h

```diff
@@ -284,6 +284,8 @@ struct usdf_ep {
 	struct usdf_tx *ep_tx;
 	struct usdf_rx *ep_rx;
 
+	struct fi_ep_attr ep_attr;
```

Comment:
Actually, should I store a copy of the whole `fi_ep_attr` struct? Maybe I should just store the `max_msg_size`...